docs/languages/_posts/2018-12-05-c.md

```diff
@@ -0,0 +1,22 @@
+---
+title: The C Programming Language
+layout: default
+last-modified: 2018-12-05
+featured-image:
+tags: [c]
+authors:
+  - the_renegade_coder
+---
+As usual, I decided to dive into the history of C with [Wikipedia][1].
```

Comment:
Can you limit line length to like 100 or so characters? I've been using Atom which does this for me for the most part, but I think it should be standard throughout the repo.